tests/lib/utils.js

```diff
@@ -65,15 +65,11 @@ describe('utils', () => {
     describe('the file does not have a valid rule (ESM)', () => {
```

Comment:
Can you make sure there are test cases covering the named export behavior for CJS? It sounds like the behavior shouldn't change for that.